vgscollect/src/main/java/com/verygoodsecurity/vgscollect/widget/SSNEditText.kt

```diff
@@ -97,6 +99,26 @@ class SSNEditText @JvmOverloads constructor(
         return getNumberDivider()
     }
 
+    /**
+     * Sets the symbol that will divide groups of digits in the number before submit it.
+     * The divider has no impact on UI.
+     * 000 00 0000
+     *
+     * @param char The divider symbol.
+     */
+    fun setOutputDivider(char: Char) {
+        setOutputNumberDivider(char.toString())
+    }
+
+    /**
+     * Return symbol that will divide groups of digits in the number before submitting on Proxy.
+     *
+     * @return divider symbol
+     */
+    fun getOutputDivider(): Char? {
```

Comment:
@Endorf As I see output divider can't be null, as we do not allow to set null as output divider and default output divider is not null to, maybe we can return not null char? or maybe we can add the ability to set output divider as null? because in current implementation we can set only zero-whitespace char

Reply:
A Developer can't set null as a divider but he should be able to remove the divider from the code.
Good catch! We'll give the ability to set null.